src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,33 @@ public override SwitchParameter UseSSL
             }
         }
 
+        /// <summary>
+        /// Gets or sets strict mode.
+        /// </summary>
+        [Experimental("PSStrictModeVersionAssignment", ExperimentAction.Show)]
+        [Parameter(ParameterSetName = InvokeCommandCommand.InProcParameterSet)]
+        public Version StrictModeVersion
```

Comment:
Please use declare the following attributes for this parameter, The `-Version` parameter of `Set-StrictMode` declares the same attributes.
```
        [ArgumentToPSVersionTransformation]
        [ValidateVersion]
```

Reply:
Null is a valid value for strictmode version...

Reply:
`Set-StrictMode -Version` doesn't accept `null`. Are we explicitly allowing null value here?

```
PS:1> Set-StrictMode -Version $null
Set-StrictMode: Cannot validate argument on parameter 'Version'. The "0.0" argument does not contain a valid PowerShell version. Supply a valid version number and then try the command again.
```

Reply:
Yes, but we need a way to turn strict mode off as well.  As I mention below, we either need to accept null value or provide another parameter to allow for this.

Reply:
[ArgumentToPSVersionTransformation] and [ValidateVersion] cannot be added here because they are private classes in InternalCommands.cs

Reply:
You can change them to be `internal`. You will need the `ArgumentToPSVersionTransformation` attribute to handle `latest` as a valid argument.
`ValidateVersion` makes sure the argument is a valid powershell version and it's not null. The former check is needed no matter what. But whether the latter check is needed or not depends on whether null value is accepted or not.